helix-term/src/commands.rs

```diff
@@ -3612,10 +3614,12 @@ fn replace_selections_with_clipboard_impl(
 
 fn replace_selections_with_clipboard(cx: &mut Context) {
     let _ = replace_selections_with_clipboard_impl(cx.editor, ClipboardType::Clipboard, cx.count());
+    exit_select_mode(cx);
```

Comment:
This could be part of replace_selections_with_clipboard_impl?

Reply:
Yes I can change this. I was originally doing that but changed to this after checkout out the `yank` command code, which seems to do exactly this.

Reply:
Moved the `exit_select_mode(cx)` call into `replace_selections_with_clipboard_impl()`. Had to also change the parameters since in needs access to `cx`.